cnames_active.js

```diff
@@ -300,6 +300,7 @@ var cnames_active = {
   "cachi": "eddiejibson.github.io/cachi",
   "caffeine": "ccrraaiigg.github.io/caffeine",
   "cagatay": "cagataycali.github.io",
+  "caissa": "https://agentx-cgn.github.io/caissa/dist/",
```

Comment:
```suggestion
  "caissa": "https://agentx-cgn.github.io/caissa/dist/",
```
We cannot CNAME to a directory, so currently users would need to go to `caissa.js.org/dist/` to view the content of `dist`. Is this intended?

Reply:
No, will update and then commit suggestion, thx for heads up.